core/src/main/java/org/apache/druid/java/util/common/io/Closer.java

```diff
@@ -123,6 +123,17 @@ private Closer()
     return closeable;
   }
 
+  /**
+   * Registers a list of  {@code closeable} to be closed when this {@code Closer} is
+   * {@linkplain #close closed}.
+   */
+  // close. this word no longer has any meaning to me.
```

Comment:
Please remove this comment, tt doesn't seem to make any sense.

Reply:
This is copy-paste. It is already in current master :-)

https://github.com/apache/incubator-druid/blob/master/core/src/main/java/org/apache/druid/java/util/common/io/Closer.java#L116

I'll remove it anyway 👍